fvtest/compilertriltest/LogicalTest.cpp

```diff
@@ -103,3 +115,66 @@ INSTANTIATE_TEST_CASE_P(LogicalTest, Int32LogicalBinary, ::testing::Combine(
         std::make_tuple("iand", iand),
         std::make_tuple("ixor", ixor)
         )));
+
+class Int64LogicalBinary : public TRTest::BinaryOpTest<int64_t> {};
+
+TEST_P(Int64LogicalBinary, UsingConst) {
+    auto param = TRTest::to_struct(GetParam());
+
+    char inputTrees[120] = {0};
```

Comment:
Hm... It looks like the length of the string needs to be increased. Maybe 160? You can try running the tests on your local machine to see if it works.